src/plugins/common/constants.js

```diff
@@ -11,6 +11,7 @@ export const serviceIds = {
 
   ADVERTISING_SUBSCRIBER: t('advertising_subscriber'),
   CUSTOM_CODE_SUBSCRIBER: t('custom_code_subscriber'),
+  HAS_RELATED_TEASERS_SUBSCRIBER: t('has_related_teasers_subscriber'),
```

Comment:
@albertgcsula this should go in the curator plugin

Reply:
@gdbrown moved.